wordpress-mobile/WordPress-iOS
# Stats: Remove statsRevampV2 feature flag

Remove statsRevampV2 feature flag and the code that falls under the disabled flag.

## To test:

I think the most important part is code review. However, quick visual review of Insights would also be good

1. Open Jetpack app
2. Stats
3. Insights
4. Confirm cells contain headers, data loads, Insight management works, navigation works

## Regression Notes
1. Potential unintended areas of impact

Affecting Stats Insights

2. What I did to test those areas of impact (or what existing automated tests I relied on)

- Compiler
- Running existing tests
- Manual testing

PR submission checklist:


UI Changes testing checklist:
